src/manifolds/SymmetricPositiveDefinite.jl

```diff
@@ -128,6 +276,109 @@ function default_estimation_method(::SymmetricPositiveDefinite, ::typeof(mean))
     return GeodesicInterpolation()
 end
 
+@doc raw"""
+    convert(::Type{AbstractMatrix}, p::SPDPoint)
+
+return the point `p` as a matrix.
+The matrix is either stored within the [`SPDPoint`](@ref) or reconstructed from `p.eigen`.
+"""
+convert(::Type{AbstractMatrix}, p::SPDPoint)
+convert(::Type{AbstractMatrix}, p::SPDPoint) = p.p
+function convert(::Type{AbstractMatrix}, p::SPDPoint{Missing})
+    return (p.eigen.vectors * Diagonal(p.eigen.values) * p.eigen.vectors')
+end
+
+@doc raw"""
+    eigvals_sqrt(p::AbstractMatrix)
```

Comment:
```suggestion
    _sqrt(p::AbstractMatrix)
```
I've taken a closer look at this again and the `eigvals` part actually doesn't make sense, this is actually square root of `p`. Sorry for a poor suggestion.

Reply:
but then we have to adapt all those occurrences again, right? I liked having a new name here to not have to use `sqrt` nor `_sqrt`, so I would prefer having a prefix still. Maybe `spd_`?

Reply:
Yes, `spd_sqrt` sounds fine.